specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3956,6 +4152,329 @@
+			"insertPreTag": {
+				"type": "string",
+				"default": " ",
+				"description": "The tag indicates the start of the merged text."
+			},
+			"insertPostTag": {
+				"type": "string",
+				"default": " ",
+				"description": "The tag indicates the end of the merged text."
+			}
+		},
+		"description": "The skill for merging the text."
+	},
+	"NamedEntityRecognitionSkill": {
+		"x-ms-discriminator-value": "#Microsoft.Skills.Text.NamedEntityRecognitionSkill",
+		"allOf": [{
+			"$ref": "#/definitions/Skill"
+		}],
+		"properties": {
+			"categories": {
+				"type": "array",
+				"items": {
+					"type": "string"
+				},
+				"description": "A list of categories."
+			},
+			"defaultLanguageCode": {
+				"type": "string",
+				"default": "en",
+				"description": "A value indicating which language code. Default is en."
+			},
+			"minimumPrecision": {
+				"type": "number",
+				"format": "double",
+				"x-nullable": true,
+				"description": "A value between 0 and 1 to indicates the confidence of the results."
+			}
+		},
+		"description": "Text analytics named entity recognition."
+	},
+	"SentimentSkill": {
+		"x-ms-discriminator-value": "#Microsoft.Skills.Text.SentimentSkill",
+		"allOf": [{
+			"$ref": "#/definitions/Skill"
+		}],
+		"properties": {
+			"defaultLanguageCode": {
+				"type": "string",
+				"default": "en",
+				"description": "A value indicating which language code. Default is en."
+			}
+		},
+		"description": "Text analytics sentiment analysis."
+	},
+	"SplitSkill": {
+		"x-ms-discriminator-value": "#Microsoft.Skills.Text.SplitSkill",
+		"allOf": [{
+			"$ref": "#/definitions/Skill"
+		}],
+		"properties": {
+			"defaultLanguageCode": {
+				"type": "string",
+				"default": "en",
+				"description": "A value indicating which language code. Default is en."
+			},
+			"textSplitMode": {
+				"type": "string",
+				"description": "A value indicating which split mode to perform. The supported values are 'pages' and 'sentences'."
+			},
+			"maximumPageLength": {
+				"type": "integer",
+				"format": "int32",
+				"x-nullable": true,
+				"description": "The desired maximum page length."
+			}
+		},
+		"description": "The skill to split text."
+	},
+	"WebApiSkill": {
+		"x-ms-discriminator-value": "#Microsoft.Skills.Custom.WebApiSkill",
+		"allOf": [{
+			"$ref": "#/definitions/Skill"
+		}],
+		"properties": {
+			"uri": {
+				"type": "string",
+				"description": "The url for the Web Api."
+			},
+			"httpHeaders": {
+				"$ref": "#/definitions/WebApiHttpHeaders",
+				"description": "The headers required to make the http request."
+			},
+			"httpMethod": {
+				"type": "string",
+				"description": "The method for the http request."
+			},
+			"timeout": {
+				"type": "string",
+				"format": "duration",
+				"description": "The desired timeout for the request."
+			},
+			"batchSize": {
+				"type": "integer",
+				"format": "int32",
+				"x-nullable": true,
+				"description": "The desired batch size."
+			}
+		},
```

Comment:
Aren't some of the above properties required?

Reply:
Yes. Added the required properties. Thanks.